mne/viz/_brain/_brain.py

```diff
@@ -504,10 +502,42 @@ def __init__(self, subject_id, hemi='both', surf='pial', title=None,
         self.plotter = self._renderer.plotter
 
         self._setup_canonical_rotation()
+        self._add_hemis(surf, alpha, offset, geo_kwargs)
+
+        self.interaction = interaction
+        self._closed = False
+        if show:
+            self.show()
+        # update the views once the geometry is all set
         for h in self._hemis:
+            for ri, ci, v in self._iter_views(h):
+                self.show_view(v, row=ri, col=ci, hemi=h)
+
+        if surf == 'flat':
+            self._renderer.set_interaction("rubber_band_2d")
+
+    def _setup_canonical_rotation(self):
```

Comment:
please move back, having it moved up here makes the diff harder to read and blame harder to follow with minimal gain

Reply:
Sure